docs/kola.md

```diff
@@ -73,6 +73,11 @@ Example format of the file:
     # This test will be skipped on these arches
     # If no arches are specified, test will be skipped on all arches
     - s390x
+  platforms:
+    # This test will be skipped on these platforms
+    # If no arches are specified, test will be skipped on all platforms
```

Comment:
```suggestion
    # If no platforms are specified, test will be skipped on all platforms
```

Reply:
oops - copy/paste error :)

I'm going to re-push and skip CI since it's just a docs change.